apps/directory/src/stores/filtersStore.ts

```diff
@@ -223,17 +223,29 @@ export const useFiltersStore = defineStore("filtersStore", () => {
   }
 
   function addOntologyOptions(filterName: string, value: IOntologyItem[]) {
+    /// Temporary fix for issue #906
+    const diagnosisavailableCount = filters.value.Diagnosisavailable?.length
+      ? filters.value.Diagnosisavailable?.length
```

Comment:
questionmark on Diagnosisavailable is not necessary since it's checked the line above.